.ci/google-java-format.sh

```diff
@@ -19,101 +40,51 @@ else
   exit 1
 fi
 
+echo "Checking that all excluded java files have same size as matching InputFormatted* file:"
+MISMATCH_CONTENT=$(cat "${EXCLUDED_LIST[@]}" \
+  | while read -r path; do
+    size=$(sed -E 's|^[[:space:]]+||; s|//.*||' "$path" | wc -c)
+    name=$(echo "$path" | sed -E 's/.*Input([^\.]+)\.java.*/\1/')
+    formatted_path=$(find ./src -type f -name "InputFormatted${name}.java")
+    size_formatted=$(sed -E 's|^[[:space:]]+||; s|//.*||' "$formatted_path" | wc -c)
+    diff=$(( size - size_formatted ))
+    if (( diff < 0 )); then
+      diff=$(( -diff ))
+    fi
+    if is_suppressed "Input${name}.java";then
```

Comment:
Add a space after ;

Reply:
This condition should be under conditions like `   if (( diff > 500 )); then` .
To skip violation if diff of speci size and it is in corresponding suppression list.
`exit 1`  should be only if not suppressed for special diff size